code/modules/mob/living/simple_animal/hostile/killertomato.dm

```diff
@@ -0,0 +1,26 @@
+/mob/living/simple_animal/hostile/killertomato
+	name = "Killer Tomato"
+	desc = "It's a horrifyingly enormous beef tomato, and it's packing extra beef!"
```

Comment:
I honestly don't see the issue, you wouldn't say "The Grizzly bear mauled the idiot that tried to hug it."